cns/requestcontroller/kubecontroller/crdrequestcontroller.go

```diff
@@ -242,7 +271,7 @@
 }
 
 // UpdateCRDSpec updates the CRD spec
-func (crdRC *crdRequestController) UpdateCRDSpec(cntxt context.Context, crdSpec nnc.NodeNetworkConfigSpec) error {
+func (crdRC *crdRequestController) UpdateCRDSpec(cntxt context.Context, crdSpec nnc.NodeNetworkConfigSpec)  error {
```

Comment:
I don't think anything changed here, just an extra space snuck in before the return type